components/CardList/CardList.styled.js

```diff
@@ -24,3 +24,21 @@ export const Input = styled.input`
 export const Results = styled.p`
   text-align: center;
 `;
+export const FilterContainer = styled.section`
+  margin: 1em;
+  display: flex;
```

Comment:
Do you really need a flexbox here? 🤔

Reply:
"I suppose I need it. I've attempted to remove it, but the container shifts to the left."